packages/blade/src/components/Dropdown/DropdownOverlay.web.tsx

```diff
@@ -34,6 +34,7 @@ const _DropdownOverlay = ({
   testID,
   zIndex = componentZIndices.dropdownOverlay,
   width,
+  placement = 'bottom-start',
```

Comment:
lets call it `defaultPlacement` or `preferredPlacement`? because placement can be anywhere depending on remaining screen size. This is just default.

Reply:
Yes, we can call it `defaultPlacement `.